src/android-emulator.js

```diff
@@ -38,12 +38,16 @@ export default AndroidEmulator;
  *
  * @param {Object} [opts] - Various options.
  * @param {Boolean} [opts.force] - When `true`, bypasses the cache and forces redetection.
- * @param {SDK} [opts.sdk] - When passed in, it will attempt to resolve the AVD's target, SDK
+ * @param {Array<SDK>} [opts.sdks] - When passed in, it will attempt to resolve the AVD's target, SDK
  * version, and API level.
  * @returns {Promise<Array<AndroidEmulator>>}
  */
 export function getEmulators({ force, sdks } = {}) {
-	return cache(`androidlib:avd:${sdks && (sdks.path || sdks[0].path) || ''}`, force, async () => {
+	let cacheString = 'androidlib:avd';
+	if (sdks) {
+		cacheString = `${cacheString}:${sdks[0] && sdks[0].path || ''}`;
```

Comment:
Since we mandate `opts.sdks` be an array, we must take into account each sdk in the cache key.

```js
return cache(`androidlib:avd:${sdks.map(s => s.path).sort().join(':')}`, force, async () => {
```

Plus we can go back to a one-liner. :)